spec/javascripts/health-care/components/form-elements/ErrorableRadioButtonGroup.spec.jsx

```diff
@@ -0,0 +1,166 @@
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `label` of type `boolean` supplied to `ErrorableRadioButtonGroup`, expected `string`./);
+    });
+
+    it('onValueChange is required', () => {
+      SkinDeep.shallowRender(<ErrorableRadioButtonGroup label="test" options={options}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Required prop `onValueChange` was not specified in `ErrorableRadioButtonGroup`/);
+    });
+
+    it('onValueChange must be a function', () => {
+      SkinDeep.shallowRender(<ErrorableRadioButtonGroup label="test" options={options} onValueChange/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `onValueChange` of type `boolean` supplied to `ErrorableRadioButtonGroup`, expected `function`/);
+    });
+
+    /*
+    it('errorMessage must be a string', () => {
+      SkinDeep.shallowRender(
+        <ErrorableRadioButtonGroup label="test" errorMessage options={options} onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `errorMessage` of type `boolean` supplied to `ErrorableRadioButtonGroup`, expected `string`/);
+    });
+    */
+
+    it('options is required', () => {
+      SkinDeep.shallowRender(
+        <ErrorableRadioButtonGroup label="test" onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Required prop `options` was not specified in `ErrorableRadioButtonGroup`/);
+    });
+
+    it('options must be an object', () => {
+      SkinDeep.shallowRender(
+        <ErrorableRadioButtonGroup label="test" options onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `options` of type `boolean` supplied to `ErrorableRadioButtonGroup`, expected an array/);
+    });
+
+    it('value must be a string', () => {
+      SkinDeep.shallowRender(
+        <ErrorableRadioButtonGroup label="test" options={options} value onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `value` of type `boolean` supplied to `ErrorableRadioButtonGroup`, expected `string`/);
+    });
+
+    it('required must be a boolean', () => {
+      SkinDeep.shallowRender(
+        <ErrorableRadioButtonGroup label="test" options={options} required="hi" onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `required` of type `string` supplied to `ErrorableRadioButtonGroup`, expected `boolean`/);
+    });
+  });
+
+  /*
+  it('ensure value changes propagate', () => {
+    let errorableRadioButtonGroup;
+
+    const updatePromise = new Promise((resolve, _reject) => {
+      errorableRadioButtonGroup = ReactTestUtils.renderIntoDocument(
+        <ErrorableRadioButtonGroup label="test" options={options} onValueChange={(update) => { resolve(update); }}/>
+      );
+    });
+
+    const input = ReactTestUtils.findRenderedDOMComponentWithTag(errorableRadioButtonGroup, 'input');
```

Comment:
We're going to have to change this test from checking for any input and instead checking for the one that's checked.